src/apps/dashboard/controllers/networking.js

```diff
@@ -46,6 +46,8 @@ function onSubmit(e) {
                     return s.length > 0;
                 });
 
+                const publicUserListing = form.querySelector('#selectPublicUserListing');
```

Comment:
I think `.value` was fine.
Otherwise, it would be better to rename `publicUserListing` to `selectPublicUserListing` to match the other function. Or vice versa.

Reply:
Great, renamed the variable and extra spacing added